roster2/tool/src/java/org/sakaiproject/roster/impl/SakaiProxyImpl.java

```diff
@@ -1363,37 +1365,59 @@ public void update(Observable o, Object arg) {
         if (arg instanceof Event) {
             Event event = (Event) arg;
             String eventName = event.getEvent();
+
+            //When a user changes the name pronunciation, the changes need to be reflected in roster
+            if(ProfileConstants.EVENT_PROFILE_NAME_PRONUN_UPDATE.equals(eventName)){
+                StopWatch watch = new StopWatch();
```

Comment:
put the stop watch in a debug conditional i.e. if (log.debugEnabled) { // stopwatch stuff }